cedar-policy-cli/src/lib.rs

```diff
@@ -94,10 +94,48 @@ pub enum Commands {
     Link(LinkArgs),
     /// Format a policy set
     Format(FormatArgs),
+    /// Translate JSON schema to natural schema syntax and vice versa (except comments)
+    TranslateSchema(TranslateSchemaArgs),
     /// Create a Cedar project
     New(NewArgs),
 }
 
+#[derive(Args, Debug)]
+pub struct TranslateSchemaArgs {
+    /// The direction of translation,
+    ///   JSON -> Human
+    ///   Human -> JSON
+    #[arg(long)]
+    pub direction: TranslationDirection,
+    /// Filename to read the schema from.
+    /// If not provided, will default to reading stdin.
+    #[arg(short, long = "schema", value_name = "FILE")]
+    pub input_file: Option<String>,
+}
+
+/// The direction of translation
+#[derive(Debug, Clone, Copy, ValueEnum)]
+pub enum TranslationDirection {
+    /// JSON -> Human schema syntax
+    JsonToHuman,
+    /// Human schema syntax -> JSON
+    HumanToJson,
+}
+
+#[derive(Debug, Clone, Copy, ValueEnum)]
+pub enum SchemaFormat {
+    /// Human-readable format
+    Human,
+    /// JSON format
+    Json,
+}
```

Comment:
did we decide Human vs Natural in the CLI interface?

Reply:
`Human` as we already called it `human` for policies